aws/cloudformation/components/ami.yml.erb

```diff
@@ -132,7 +134,7 @@ frontend_device_name ||= '/dev/sda1'
         WaitOnResourceSignals: true
     Properties:
       AutoScalingGroupName: !Sub "Frontends-${AWS::StackName}"
-      VPCZoneIdentifier: <%= subnets.to_json %>
+      VPCZoneIdentifier: <%= frontend_subnets.to_json %>
```

Comment:
[Update requires some interuptions](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-autoscaling-autoscalinggroup.html#cfn-autoscaling-autoscalinggroup-vpczoneidentifier).

> When you update VPCZoneIdentifier, this retains the same Auto Scaling group and replaces old instances with new ones, according to the specified subnets.

However, this is no-op because the resolved value of `frontend_subnets` is unchanged (b,c,d)